types/twitter_cldr/index.d.ts

```diff
@@ -0,0 +1,173 @@
+// Type definitions for twitter_cldr 2.2
+// Project: https://github.com/twitter/twitter-cldr-npm/
+// Definitions by: lionel-rowe <https://github.com/DefinitelyTyped>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.9
+
+export type CldrData = Record<string, any>;
+
+export type Locale = 'af' | 'ar' | 'be' | 'bg' | 'bn' | 'ca' | 'core' | 'cs' | 'cy' | 'da' | 'de-CH' | 'de' | 'el' | 'en-150' | 'en-AU' | 'en-CA' | 'en-GB' | 'en-IE' | 'en-SG' | 'en-ZA' | 'en'
+	| 'es-419' | 'es-CO' | 'es-MX' | 'es-US' | 'es' | 'eu' | 'fa' | 'fi' | 'fil' | 'fr-BE' | 'fr-CA' | 'fr-CH' | 'fr' | 'ga' | 'gl' | 'he' | 'hi' | 'hr' | 'hu' | 'id' | 'is' | 'it-CH' | 'it' | 'ja'
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/twitter_cldr/index.d.ts;
  Line: 10;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.